code/lib/codemod/src/transforms/__tests__/migrate-to-test-package.test.ts

```diff
@@ -0,0 +1,46 @@
+import { expect, test } from 'vitest';
+import transform from '../migrate-to-test-package';
+import dedent from 'ts-dedent';
+
+expect.addSnapshotSerializer({
+  serialize: (val: any) => (typeof val === 'string' ? val : val.toString()),
+  test: () => true,
+});
+
+const tsTransform = async (source: string) =>
+  (await transform({ source, path: 'Component.stories.tsx' })).trim();
+
+test('replace jest and testing-library with the test package', async () => {
+  const input = dedent`
+    import { expect } from '@storybook/jest';
+    import { within, userEvent } from '@storybook/testing-library';
+  `;
+
+  expect(await tsTransform(input)).toMatchInlineSnapshot(`
+    import { expect } from '@storybook/test';
+    import { within, userEvent } from '@storybook/test';
```

Comment:
I guess merging both lines is possible but might require some further action?

Reply:
Yeah, I think people that care about this, will have a eslint rule for merging this automatically.